Doesn't seem to be working?

Pad URL http://meh.com/p/comment_foo

```
[2013-04-06 01:22:53.233] [ERROR] console - TypeError: Cannot call method 'replace' of undefined
    at prepareText (/home/jose/etherpad-lite/node_modules/ep_defaultPadText/index.js:22:15)
    at Object.exports.padCreate [as hook_fn] (/home/jose/etherpad-lite/node_modules/ep_defaultPadText/index.js:14:14)
    at normalizedhook (/home/jose/etherpad-lite/src/static/js/pluginfw/hooks.js:15:27)
    at hookCallWrapper (/home/jose/etherpad-lite/src/static/js/pluginfw/hooks.js:21:14)
    at exports.callAll (/home/jose/etherpad-lite/src/static/js/pluginfw/hooks.js:76:14)
    at Array.map (native)
    at Function._.map._.collect (/home/jose/etherpad-lite/src/node_modules/underscore/underscore.js:95:56)
    at Object.exports.callAll (/home/jose/etherpad-lite/src/static/js/pluginfw/hooks.js:75:24)
    at Pad.appendRevision (/home/jose/etherpad-lite/src/node/db/Pad.js:92:11)
    at Object.Pad.init (/home/jose/etherpad-lite/src/node/db/Pad.js:399:13)
```

Settings.json

```
  "ep_defaultPadText":{
    "comment_" : {text:"Comment.."}
  }
```

--- a/index.js
+++ b/index.js
@@ -19,13 +19,13 @@ exports.padCreate = function(hook, context)
 
 function prepareText(text, number)
 {
-  text = text.replace("$num$", number) || text;
+  text = text.replace("$num$", number);
   
   // Extract the $date$-placeholder out of our template
   var dateTokenRegex = /\$date:([^$]+)\$/;
   var dateToken = dateTokenRegex.exec(text);
   // replace the full placeholder by the current date formatted using the given format
-  text = text.replace(dateToken[0], strftime(dateToken[1]));
-  
+  if(dateToken) text = text.replace(dateToken[0], strftime(dateToken[1]));
+
   return text;
 }

--- a/index.js
+++ b/index.js
@@ -19,7 +19,7 @@ exports.padCreate = function(hook, context)
 
 function prepareText(text, number)
 {
-  text = text.replace("$num$", number);
+  text = text.replace("$num$", number) || text;
   
   // Extract the $date$-placeholder out of our template
   var dateTokenRegex = /\$date:([^$]+)\$/;

